.clj-kondo/config.edn

```diff
@@ -511,6 +513,10 @@
     metabase.timeline              :any
     metabase.types                 #{metabase.util}
     metabase.upload                :any
+    metabase.user-key-value        #{metabase.api
+                                     metabase.config
+                                     metabase.lib    ; WHY
```

Comment:
Totally fair to ask WHY here, I think I pulled it in solely for `:metabase.lib.schema.literal/string.datetime` and it would probably be cleaner to just copy that in here (or even better, put it in a common place?).